nuitka/specs/BuiltinParameterSpecs.py

```diff
@@ -267,6 +267,7 @@ def makeBuiltinExceptionParameterSpec(exception_name):
 builtin_len_spec = BuiltinParameterSpecNoKeywords("len", ("object",), 0)
 builtin_dict_spec = BuiltinParameterSpec("dict", (), 0, "list_args", "dict_args")
 builtin_len_spec = BuiltinParameterSpecNoKeywords("len", ("object",), 0)
+builtin_any_spec = BuiltinParameterSpecNoKeywords("any", ("object",), 0)
```

Comment:
The "any" is special in that is a function that takes varargs.

Reply:
I am little confused. I read the documentation for the `len` and `any` and found that they takes similar arguments.

Reply:
You are correct @bksahu and I was wrong, which I realized and changed the comment to say "not true:".

Somehow I was thinking of `any(1,2,3)` being a thing, but it is rather `any([1,2,3])` or the `any(x for x in something)`.